src/main/java/core/basesyntax/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterService;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriterServiceImpl implements WriterService {
+    @Override
+    public void createReportFile(String fruitBalanceReport) {
+        File file = new File("src/main/resources/report.txt");

```

Comment:
Replace the hardcoded file path with the output file name parameter passed to the 'createReportFile' method.